### PR TITLE
Export account statement

### PR DESCRIPTION
**Pull request by soerenbf**
## Purpose

Add account statement page linking to ccdscan under account settings.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: packages/browser-wallet/src/popup/pages/Account/AccountSettings/AccountStatement.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    const account = useAtomValue(selectedAccountAtom);
+    const url = useMemo(
+        () => `${isMainnet(network) ? urls.ccdscanMainnet : urls.ccdscanTestnet}${SELECT_ACCOUNT_PATH}${account}`,
+        [network, account]
+    );
+
+    return (
+        <div className="flex-column align-center white-space-break justify-space-between m-10 text-center">
+            <div>{t('description')}</div>
+            <Button width="wide" as={ExternalLink} path={url}>
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/pages/Account/AccountSettings/AccountStatement.tsx:25]: Is the text supposed to be dark?

Screenshot from 2022-12-08 16-36-34

**Review by shjortConcordium — COMMENTED** [review]
I think the background should be white, like it is for the export key page. 
And the page looks a little weird with the top aligned text, but no title :thinking:

**Review by shjortConcordium — APPROVED** [review]
Text still dark for me :thinking: 
Screenshot from 2022-12-09 10-46-55
but otherwise LGTM

File: packages/browser-wallet/src/popup/pages/Account/AccountSettings/AccountStatement.tsx (hunk 1 above)
Comment (soerenbf) [reply packages/browser-wallet/src/popup/pages/Account/AccountSettings/AccountStatement.tsx:28]: Was due to a missing `:visited` rule on buttons. Fixed now.

**Pull request merged**